packages/manager/apps/web/client/app/components/manager-order-catalog-price/index.js

```diff
@@ -0,0 +1,11 @@
+import angular from 'angular';
+import component from './manager-order-catalog-price.component';
+
+const moduleName = 'ovhManagerOrderCatalogPrice';
+
+angular
+  .module(moduleName, [])
```

Comment:
```suggestion
  .module(moduleName, [`oui`, `angular-translate`])
```
?

Reply:
Hmm injecting `angular-translate` causes error, even with `angular-translate` imported

Reply:
Legit ! angular's module name is `pascalprecht.translate` 😉